### ISSUE TITLE
[Usable] 'Mic' button is not accessible in scan mode.

### ISSUE THREAD

**Issue by priysn** [Microsoft/BotFramework-WebChat#1410]
**Actual Result:**
'Mic' button is not accessible in scan mode.

**Expected Result:**
'Mic' button should be accessible in scan mode.

**Repro Steps:**

1. Open URL [https://apac01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fmicrosoft.github.io%2FBotFramework-WebChat%2Fspeech-cognitive-services-bing-speech%2F%3Fs%3D7c7dacfe04044f4f99a1edcf9b0e2ab9&data=02%7C01%7Cv-saugup%40microsoft.com%7C0f6f5e63167449daa46d08d65570f676%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636790341026155420&sdata=gcFq8JsflCLfUKAyw8f2A19aJcEQEFxGzjASMK9S59A%3D&reserved=0](https://apac01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fmicrosoft.github.io%2FBotFramework-WebChat%2Fspeech-cognitive-services-bing-speech%2F%3Fs%3D7c7dacfe04044f4f99a1edcf9b0e2ab9&data=02%7C01%7Cv-saugup%40microsoft.com%7C0f6f5e63167449daa46d08d65570f676%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636790341026155420&sdata=gcFq8JsflCLfUKAyw8f2A19aJcEQEFxGzjASMK9S59A%3D&reserved=0) in Edge browser.
2. Notice 'Upload File' and 'Mic' button icons present in bottom of screen.
3. Open Narrator (Using Ctrl + Windows +Enter).
4. Start Scan mode (Caps lock + Space).
5. Navigate to Mic button by using Up/Down arrow keys.
6. Observe that 'Mic' button is not accessible in scan mode.
 
**User Impact:**
Users who rely on screen readers may get confused and face difficulty if navigation is not correct or logical and controls are not accessible in scan mode.

**Test Environment:**
**OS:** Windows 10
**OS Build:** 17134.345
**OS Version:** 1803
**Browser:** Edge
**Tool:** Narrator

**Attachment:**
[Usable_Mic button not accessible in scan mode.zip](https://github.com/Microsoft/BotFramework-WebChat/files/2628554/Usable_Mic.button.not.accessible.in.scan.mode.zip)

**Comment by corinagum** [comment]
Related to #1307

**Comment by corinagum** [comment]
Unable to repro

**Comment by Amit8527510735** [comment]
#HCL;#Accessibility;#HCL_BotFramework_WebChat;A11yUsable
